services/analyzer/sequelize-tables-analyzer.js

```diff
@@ -371,6 +371,12 @@ async function analyzeSequelizeTables(databaseConnection, config, allowWarning)
   const referencesPerTable = createAllReferences(databaseSchema, schemaAllTables);
   Object.keys(referencesPerTable).forEach((tableName) => {
     schemaAllTables[tableName].references = _.sortBy(referencesPerTable[tableName], 'association');
+
+    if (!schemaAllTables[tableName].fields.length) {
```

Comment:
[MISC] You could add a comment here explaining why you're doing this

Reply:
Fixed